### PR TITLE
Add solution to Two Fer.

### PR DESCRIPTION
**Pull request by rudresh4**
## Why?
Solve unimplemented Two Fer problem for Ruby.

## What?
Two Fer problem description: https://github.com/exercism/problem-specifications/blob/master/exercises/two-fer/description.md

## How Has This Been Tested?
rake test
rake test:two-fer

## Types of changes
- [x] Add Two Fer solution
- [x] Add Two Fer test case
- [x] Update config.json

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -176,6 +176,16 @@
...
+      "uuid": "8d345e34-e815-49cd-91ab-ca8db63b07ef",
+      "slug": "two-fer",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+
+      ]
+    },
+    {
```
Comment (Insti) [inline config.json]: Please put new exercises at the end of the list.

Is this difficulty assessment accurate?

Please add some topics.

File: exercises/two-fer/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    gem install minitest
+
+If you would like color output, you can `require 'minitest/pride'` in
+the test file, or note the alternative instruction, below, for running
+the test file.
+
+In order to run the test, you can run the test file from the exercise
+directory. For example, if the test suite is called
+`hello_world_test.rb`, you can run the following command:
```
Comment (Insti) [inline exercises/two-fer/README.md]: This file can include the actual instructions.

**Comment by Insti** [comment]
Thanks @rudresh4 this is a great start.

I'd like to see a test generator added which would automatically generate the tests based on the [common test data](https://github.com/exercism/problem-specifications/blob/master/exercises/two-fer/canonical-data.json). See the instructions about this in the [README](https://github.com/exercism/ruby#implementing-a-generator). The generator for this problem should not be difficult to implement.

**Comment by rudresh4** [comment]
@Insti Updates made as requested.

**Comment by Insti** [comment]
Awesome :heart: thanks.
Can you also commit the generator please?

File: exercises/two-fer/.meta/.version [hunk 3]
```diff
@@ -0,0 +1 @@
+3
```
Comment (Insti) [inline exercises/two-fer/.meta/.version]: Change the version back to 1.

To do this, locally change this number to `0` then re-generate the tests, which should update it to 1 everywhere.

File: exercises/two-fer/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+    gem install minitest
+
+If you would like color output, you can `require 'minitest/pride'` in
+the test file, or note the alternative instruction, below, for running
+the test file.
+
+In order to run the test, you can run the test file from the exercise
+directory. For example, if the test suite is called
+`two_fer_test.rb`, you can run the following command:
```
Comment (Insti) [inline exercises/two-fer/README.md]: > For example, if the test suite is called `two_fer_test.rb`, you can run the following command:

Can probably all be deleted.

**Comment by Insti** [comment]
Looking good we're almost there 👍

**Comment by rudresh4** [comment]
@Insti All updates are made based on request

File: exercises/two-fer/README.md [hunk 5]
```diff
@@ -14,24 +14,5 @@ If no name is given, the result should be "One for you, one for me."
 For installation and learning resources, refer to the
 [exercism help page](http://exercism.io/languages/ruby).
 
-For running the tests provided, you will need the Minitest gem. Open a
```
Comment (Insti) [inline exercises/two-fer/README.md]: I think this section on how to run the tests is required, it was just the "For example.." line that I thought needed to be removed.

Sorry if my earlier request was confusing.

Comment (rudresh4) [reply exercises/two-fer/README.md]: I solved this problem once and for all by autogenerating READMEs: https://github.com/exercism/ruby/pull/721

Comment (Insti) [reply exercises/two-fer/README.md]: Ok, the testing instructions will need to be part of the README in this PR though.

**Comment by Insti** [comment]
@rudresh4 why did you close all your PRs? 😞

**Pull request closed without merging**